hazelcast/pom.xml

```diff
@@ -724,7 +724,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>42.3.3</version>
+            <version>${postgresql.version}</version>
```

Comment:
I think you should get rid of the version specification completely here as you already have it in the `dependencyManagement`.

Reply:
I will send a separate PR to fix this across all dependencies.